### DISCUSSION TITLE
Madouqu does not enabled unless rebuild metatube-server

### DISCUSSION THREAD

**Discussion by charleshuang3** [metatube-community/jellyfin-plugin-metatube#395] [Q&A] [answered]
I found some providers for example Madouqu does not enable by default, not added to register.

https://github.com/metatube-community/metatube-sdk-go/blob/16ab1ea6e1bdd69fc687117da2518f76853889d2/engine/register.go

Is it intended. Should we fix? I am happy to send a PR for this.

**Comment by xjasonlyu** [comment, marked as answer]
Yep, this is intended. Madouqu is currently disabled due to stability issues. Also, it's the only non-Japanese provider, so I disabled it to prevent mixed languages in a single media library..

**Reply by charleshuang3** [reply]
Thank you for reply. Also want to ask if metatube accept pr to scrape from https://modelmediaasia.com/ or other non jav site

**Reply by xjasonlyu** [reply]
That's on the roadmap, but have to wait until the metatube ecosystem supports multi-region content.
